docs/share.md

```diff
@@ -12,19 +12,28 @@ You can share the data your connector accesses by connecting to the data, creati
 
 Or, if you prefer to have your users test the connector, you can place the connector files in the directories and then run the commands as follows:
 
-**Tableau Desktop:** This is the same as developing the connector.
-Place the files into a subdirectory you create, and then use the -DConnectPluginsPath command line argument.  For example, place your plugin files in `C:\tableau_connectors\myconnector` and then start Tableau:
+**Tableau Desktop:** 
 
-```
-tableau.exe -DConnectPluginsPath=C:\tableau_connectors
-```
+1. Create a directory for Tableau connectors. For example: `C:\tableau_connectors`
+1. Put the the folder containing your connector's manifest.xml file in this directory. Each connector should have its own folder. For example: `C:\tableau_connectors\my_connector`
+1. Run Tableau using the `-DConnectPluginsPath` command line argument, pointing to your connector directory. For example: 
 
-**Tableau Server:** Follow the same instructions as Tableau Desktop for each server node.
-Then set this option: "native_api.connect_plugins_path: C:/tableau_connectors".
-For information about using TSM to do this, see [tsm configuration set Options](https://onlinehelp.tableau.com/current/server-linux/en-us/cli_configuration-set_tsm.htm) in the Tableau Server help.
+    ```
+    tableau.exe -DConnectPluginsPath=C:\tableau_connectors
+    ```
 
-For example:
+**Tableau Server:** 
 
-```
-tsm configuration set -k native_api.connect_plugins_path -v C:/tableau_connectors
-```
+1. For each server node, follow Tableau Desktop's steps 1 and 2 above.
+1. Set the `native_api.connect_plugins_path` option. For example:
+
+    ```
+    tsm configuration set -k native_api.connect_plugins_path -v C:/tableau_connectors 
+    ```
+  
+1. Restart Tableau Server to see the new connector.
```

Comment:
Looks pretty good! Just one last thing.

It's mentioned in the linked tsm help page, but to apply the changes you need to run the proper command, which automatically restarts the server. This isn't a tutorial on tsm, and we don't want it to be, so walking the reader through applying the change isn't necessarily. The upshot is that for step 2, they will have already restarted the server, so another restart is redundant.

My initial comment on restarting the server was imagining it as a comment outside of the steps, since it will come up not when someone is initially trying to get their connectors working on a server but when they update their connectors and want to see their changes reflected on the server. Maybe something like:
"Updating, removing or adding connectors will require a server restart before the changes show up on the server."

Reply:
Got it. I removed the step and added a note about restarting the server when changes are made.